app/components/common/FinalFooter.vue

```diff
@@ -34,7 +69,10 @@ export default Vue.extend({
     picture(v-else)
       source(srcset="/images/pages/base/logo.webp" type="image/webp")
       img(src="/images/pages/base/logo.png" alt="CodeCombat logo")
-    .float-right
+    .social-links
+      a(v-for="socialLink in socialLinks" :href="socialLink.href" :key="socialLink.href" target="_blank")
+        img(:src="socialLink.img" :alt="socialLink.alt")
+    .copyright
```

Comment:
Are these social links showing when it's in China mode? Might need to check that they aren't.